lib/tasks/telemetry.rake

```diff
@@ -0,0 +1,47 @@
+require "foreman/telemetry"
```

Comment:
Is this rake task simply for printing metrics to a console by a user on the box?

Reply:
One task is just to print them, the other one is actually to generate statsd_exporter configuration so we can use statsd endpoint and still integrate it with Prometheus: https://github.com/prometheus/statsd_exporter